code/style.css

```diff
@@ -1,4 +1,379 @@
+    margin: auto; 
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.main-article-child img {
+    width: 100%;
+    border: 10px solid white;
+    box-sizing: border-box;
+}
+
+.main-article-header {
+    border-bottom: solid 1px #214273;
+    padding: 0px 0px 10px 0px;
+    box-sizing: border-box;
+    margin: 0;
+}
+
+.main-article-child div h2 a {
+    color: #214273;
+    text-decoration: none;
+}
+
+.main-article-child div h2 a:hover {
+    color: #fa5aaa;
+}
+
+.text-area {
+    background: white;
+    margin-top: 10px;
+    padding: 15px 15px;
+}
+
+ .text-area p {
+    margin: auto;
+    padding: 10px 0px;
+    box-sizing: border-box;
+}
+
+.read-more a{
+    background: #ff8cc5;
+    text-decoration: none;
+    border-radius: 2px;
+    font-family: 'Montserrat', sans-serif; 
+    font-size: 10px;
+    font-weight: 600;
+    color:  #214273;
+    margin: 0 0;
+    padding: 5px;
+    box-sizing: border-box;
+}
+
+.read-more a:hover {
+    background: #fda7d2;
+    color: white;
+}   
+
+/* DIVIDER */
+
+.opn-divider {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 10px;
+}
+
+.opn-divider h1 {
+    font-size: 23px;
+    padding-bottom: 5px;
+    margin-bottom: 10px;
+    border-bottom: solid 1px #214273;
+    box-sizing: border-box;
+}
+
+.opn-divider img {
+    width: 40px;
+}
+
+/* OTHER ARTICLES */
+
+.other-article-parent {
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    padding: 10px 0px;
+}
+
+.other-article-child {  
+    width: 90%;
+    padding: 14px;
+    box-sizing: border-box;  
+    margin: 20px auto;  
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.other-article-child img {
+    width: 100%;
+    box-shadow: 7px 7px gainsboro;
+}
+
+.other-article-child h2 {
+    border-bottom: solid 1px #214273;
+    padding: 15px 0px 10px 0px;
+    margin: 0;
+    font-size: 20px;
+    text-align: center;
+}
+
+.other-article-child h2 a {
+    color: #214273;
+    text-decoration: none;
+}
+
+.other-article-child h2 a:hover {
+    color: #fa5aaa;
+}
+
+.other-article-child p {
+   background: #ffd7de;
+   padding: 10px 15px;
+}
+
+.read-more a{
+    background: #ff8cc5;
+    text-decoration: none;
+    border-radius: 2px;
+    font-family: 'Montserrat', sans-serif; 
+    font-size: 10px;
+    font-weight: 600;
+    color:  white;
+    margin: 0 0;
+    padding: 5px;
+    box-sizing: border-box;
+}
+
+.read-more a:hover {
+    background: #fda7d2;
+    color: white;
+} 
+
+/* FOOTER */
+
+.footer-title {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 10px;
+    box-sizing: border-box;
+}
+
+.footer-title h1 {
+    font-size: 23px;
+    padding-bottom: 5px;
+    margin-bottom: 10px;
+    border-bottom: solid 1px #214273;
```

Comment:
This works but til next time the standard is to write pixel style color - in this case: 1px solid #214273;
Just a minor detail but you've done such a good job that you're forcing us to nitpick 😅